assets/src/blocks/ENForm/FormGenerator.js

```diff
@@ -180,7 +180,7 @@ const CheckboxGen = ({field, onInputChange, onBlur, index, dependent_field, erro
             name={ name }
             type="checkbox"
             className={ `en__field__input en__field__input--checkbox ${field.name == dependent_field ? 'dependency-' + field.name : '' } ${has_error ? 'is-invalid' : ''}`}
-            defaultValue={ question_option.option_value }
+            defaultValue={ question_option.option_value || 'Y' }
```

Comment:
If we need more changes before we can move the block out of beta, it's better to address these in a separate PR that we can merge before replacing the old version of the block.